From c55548f16be7db969c4c08a23e4faf63db858635 Mon Sep 17 00:00:00 2001
From: Mark Enriquez <enriquezmark36@gmail.com>
Date: Tue, 13 Nov 2018 19:39:47 +0800
Subject: [PATCH 7/8] libion_sprd: link to libion.so instead of renaming it

This was a residue from the kanas support patchset. Wished that I
saw this earlier.

Change-Id: If3247396a5d68351275c527b8381fa5072cb5d1e
Signed-off-by: Mark Enriquez <enriquezmark36@gmail.com>
---
 libion_sprd/scx15/Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libion_sprd/scx15/Android.mk b/libion_sprd/scx15/Android.mk
index 08b324f..5ed31d6 100644
--- a/libion_sprd/scx15/Android.mk
+++ b/libion_sprd/scx15/Android.mk
@@ -52,7 +52,7 @@ LOCAL_SHARED_LIBRARIES := \
 
 LOCAL_POST_INSTALL_CMD := \
 	$(hide) mkdir -p $(TARGET_OUT_SHARED_LIBRARIES); \
-	mv -f $(TARGET_OUT_SHARED_LIBRARIES)/$(LOCAL_MODULE).so $(TARGET_OUT_SHARED_LIBRARIES)/libion.so
+	ln -sf $(LOCAL_MODULE).so $(TARGET_OUT_SHARED_LIBRARIES)/libion.so
 
 include $(BUILD_SHARED_LIBRARY)
 
-- 
2.19.1

